proxmoxer/core.py

```diff
@@ -97,21 +97,13 @@ def _request(self, method, data=None, params=None):
 
         if resp.status_code >= 400:
             if hasattr(resp, 'reason'):
-                if resp.json().get('errors') != None:
-                    raise ResourceException(
-                        resp.status_code,
-                        httplib.responses.get(resp.status_code,
-                                            ANYEVENT_HTTP_STATUS_CODES.get(resp.status_code)),
-                        resp.reason,
-                        resp.json()["errors"]
-                    )
-                else:
-                    raise ResourceException(
-                        resp.status_code,
-                        httplib.responses.get(resp.status_code,
-                                            ANYEVENT_HTTP_STATUS_CODES.get(resp.status_code)),
-                        resp.reason
-                    )
+                raise ResourceException(
```

Comment:
We don't need to check for the existence of errors here. If there are no errors, the later `.get('errors')` will pass `None` to the ResourceException which will ignore it.